doc/ref/modules/all/salt.modules.kernelpkg.rst

```diff
@@ -0,0 +1,19 @@
+.. _virtual-pkg:
```

Comment:
This tag already exists in the `doc/ref/modules/all/salt.modules.pkg.rst` file. The page tag needs to be unique. How about something like `_virtual-kernel-pkg`?